platform/macos/display_server_macos.mm

```diff
@@ -1840,9 +1840,9 @@
 	if (!p_callback.is_null()) {
 		Variant button = button_pressed;
 		Variant *buttonp = &button;
-		Variant ret;
+		Variant fun_ret;
```

Comment:
Is this one still needed now that we turned off warnings for shadowing system global stuff? `ret` is really common for this in the codebase.

Reply:
This is because of a "ret" above, so it's a local shadowing